DESCRIPTION.md

```diff
@@ -9,7 +9,25 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 # Release Notes
 
 - v3.0.5(TBD)
-  - Added the ability to read Snowflake's central configuration file.
+  - Added a feature that lets you add connection definitions to the `config.toml` configuration file. A connection definition refers to a collection of connection parameters. The connection configuration name must begin with **connections**, similar to the following that defines the parameters for the `prod` connection:
```

Comment:
It looks like we need to fix the indentation so the new content looks like"

- Added a feature that...
  
    code example

    By default...

      - On Linux
      - On Mac

    You can ...

     code example

- Bumped cryptography...
- Improved OCSP...

Reply:
Fixed the indentation, great catch! 👍